language/pt_br/passwordstrength.php

```diff
@@ -35,9 +35,9 @@
 // in a url you again do not need to specify an order e.g., 'Click %sHERE%s' is fine
 
 $lang = array_merge($lang, array(
-	'PS_VERY_WEAK'		=> 'Muito Fraca 🙁',
-	'PS_WEAK'			=> 'Fraca 🙁',
-	'PS_GOOD'			=> 'Boa 😐',
+	'PS_VERY_WEAK'		=> 'Muito Fraca 🙝',
+	'PS_WEAK'			=> 'Fraca 🙝',
+	'PS_GOOD'			=> 'Boa 😝',
```

Comment:
why are you changing the smilies?

Reply:
I guess it was a problem of utf conversion, my bad